infogami/infobase/_dbstore/save.py

```diff
@@ -97,7 +97,7 @@ def index(key, value):
                 for v in value:
                     index(key, v)
 
-        for k, v in data.iteritems():
+        for k, v in data.items():
```

Comment:
Used only once.  Python 2 will build a list in memory which will take up more RAM.  Python 3 will be fast.